vulnerabilities/importers/osv.py

```diff
@@ -221,21 +242,32 @@ def get_fixed_versions(fixed_range, raw_id) -> List[Version]:
 
     fixed_range_type = fixed_range["type"]
 
-    for version in extract_fixed_versions(fixed_range):
+    version_class = None
+    try:
+        version_class = RANGE_CLASS_BY_SCHEMES[supported_ecosystem].version_class
```

Comment:
Don't use try here, instead:
```suggestion
        version_range_class = RANGE_CLASS_BY_SCHEMES.get(supported_ecosystem)
        version_class = version_range_class.version_class if version_range_class else None
```

Reply:
Done